cms/apps/media/templates/admin/media/file/change_form.html

```diff
@@ -7,6 +7,10 @@
   {% endif %}
 {% endblock %}
 
+{% block after_related_objects %}
```

Comment:
You should add a guard here; I do believe `change_form.html` is used when adding a new instance.